fern/docs/pages/overview/quickstart.mdx

```diff
@@ -32,6 +32,22 @@ being used
 http://localhost:8001/
 ```
 
+You like one-liners, have python3.11 installed, and you are running a UNIX (MacOS & Linux) system?
+You can run it on **CPU** in few lines:
+```bash
+git clone https://github.com/imartinez/privateGPT && cd privateGPT && \
+python3.11 -m venv .venv && source .venv/bin/activate && \
+pip install --upgrade pip poetry && poetry install --with ui,local && ./scripts/setup
+
+# Launch the privateGPT API server **and** the gradio UI
+python3.11 -m private_gpt
+
+# In another terminal, create a new browser window on your private GPT!
+open http:////127.0.0.1:8001/
+```
+
+**You want to run it on GPU(s)?** Please check the more detailed [installation guide](/installation).
+
 ## API
```

Comment:
This section should probably be removed from here. It doesn't make sense here.

Reply:
The `##API` section? I agree